migrate_pim/index.rst

```diff
@@ -34,9 +34,11 @@ Then clean the cache, re-install assets and warmup the cache:
 
 .. code-block:: bash
 
+    /etc/init.d/php7.1-fpm restart
```

Comment:
sudo systemctl restart php7.1-fpm

There is no more init.d by default for this in Debian 9 and Ubuntu 16.04+

Reply:
service php7.1-fpm restart is the right way to do it. Works with init.d and systemctl based OS.

Reply:
Also, you need to indicate that the queue daemon must be stopped before doing that, or it may regenerates obsolete cache entries.